VAMobile/src/store/slices/notificationSlice.ts

```diff
@@ -49,6 +50,7 @@ export const registerDevice =
   (deviceToken?: string, refreshNotificationScreen?: boolean): AppThunk =>
   async (dispatch) => {
     dispatch(dispatchStartRegisterDevice())
+    setAnalyticsUserProperty(UserAnalytics.vama_uses_notifications(deviceToken ? true : false))
```

Comment:
This isn't the right spot for this, in retrospect I don't think. We'll need to tinker more with where this gets setup. @Sparowhawk hit me tomorrow when you're in about this.

Reply:
Should this be called in the [register](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/components/NotificationManager/NotificationManager.tsx#L22) function of the `NotificationManager` instead? `registerDevice` is called in different scenarios, so if we only want to track whether a user uses push notification once during the initial registration, I believe this would need to be moved to the `NotificationManager`

Reply:
Ya, no, Dylan has this correct where it is. I talked it out with him a lot earlier.